docs/framework/wcf/extending/client-channel-factories-and-channels.md

```diff
@@ -18,11 +18,11 @@ A channel factory creates channels.
   
 -   The <xref:System.ServiceModel.Channels.CommunicationObject> class implements <xref:System.ServiceModel.ICommunicationObject> and enforces the state machine described in step 2 of [Developing Channels](../../../../docs/framework/wcf/extending/developing-channels.md).  
   
--   The``<xref:System.ServiceModel.Channels.ChannelManagerBase> class implements <xref:System.ServiceModel.Channels.CommunicationObject> and provides a unified base class for <xref:System.ServiceModel.Channels.ChannelFactoryBase?displayProperty=nameWithType> and <xref:System.ServiceModel.Channels.ChannelListenerBase?displayProperty=nameWithType>. The <xref:System.ServiceModel.Channels.ChannelManagerBase> class works in conjunction with <xref:System.ServiceModel.Channels.ChannelBase>, which is a base class that implements <xref:System.ServiceModel.Channels.IChannel>.  
+-   The xref:System.ServiceModel.Channels.ChannelManagerBase class implements xref:System.ServiceModel.Channels.CommunicationObject and provides a unified base class for xref:System.ServiceModel.Channels.ChannelFactoryBase?displayProperty=nameWithType and xref:System.ServiceModel.Channels.ChannelListenerBase?displayProperty=nameWithType. The xref:System.ServiceModel.Channels.ChannelManagerBase class works in conjunction with xref:System.ServiceModel.Channels.ChannelBase, which is a base class that implements xref:System.ServiceModel.Channels.IChannel. 
   
--   The``<xref:System.ServiceModel.Channels.ChannelFactoryBase> class implements <xref:System.ServiceModel.Channels.ChannelManagerBase> and <xref:System.ServiceModel.Channels.IChannelFactory> and consolidates the `CreateChannel` overloads into one `OnCreateChannel` abstract method.  
+-   The xref:System.ServiceModel.Channels.ChannelFactoryBase class implements xref:System.ServiceModel.Channels.ChannelManagerBase and xref:System.ServiceModel.Channels.IChannelFactory and consolidates the CreateChannel overloads into one `OnCreateChannel` abstract method. 
   
--   The``<xref:System.ServiceModel.Channels.ChannelListenerBase> class implements <xref:System.ServiceModel.Channels.IChannelListener>. It takes care of basic state management.  
+-   The xref:System.ServiceModel.Channels.ChannelListenerBase class implements xref:System.ServiceModel.Channels.IChannelListener. It takes care of basic state management.
```

Comment:
I think the right fix here is to remove the ``` `` ```, but keep the angle brackets as they are. Otherwise the `xref`s won't work.